helpers/hgit.py

```diff
@@ -888,21 +883,18 @@ def get_path_from_git_root(
     return ret
 
 
+# TODO(gp): Just do a find.
 @functools.lru_cache()
 def get_amp_abs_path() -> str:
     """
     Return the absolute path of `amp` dir.
     """
     repo_sym_name = get_repo_full_name_from_client(super_module=False)
-    _LOG.debug("repo_sym_name=%s", repo_sym_name)
+    _LOG.info("repo_sym_name=%s", repo_sym_name)
     #
     repo_sym_names = ["alphamatic/amp"]
-    code = "get_extra_amp_repo_sym_name()"
-    try:
-        repo_sym_names.append(henv.execute_repo_config_code(code))
-    except NameError:
-        _LOG.debug("Can't execute the code '%s'", code)
-    _LOG.debug("repo_sym_names=%s", repo_sym_names)
+    repo_sym_names.append(hrecouti.get_repo_config().get_extra_amp_repo_sym_name())
```

Comment:
Nit I tend to do it in 2 lines to make it more readable.

```
extra_amp_repo_sym_name = hrecouti.get_repo_config().get_extra_amp_repo_sym_name())
repo_sym_names.append(extra_amp_repo_sym_name)
```

Probably you did a search & replace so you wanted to keep it in one line.